returnn/flow.py

```diff
@@ -0,0 +1,128 @@
+from sisyphus import tk
+from sisyphus.delayed_ops import DelayedJoin
+from typing import Optional, List, Union
+
+from i6_core import rasr
+from i6_core.returnn.training import Checkpoint
+
+
+def make_precomputed_hybrid_tf_feature_flow(
+    tf_graph: tk.Path,
+    tf_checkpoint: Checkpoint,
+    feature_tensor_name: str = "data",
+    output_layer_name: str = "output",
+    native_ops: Optional[Union[tk.Path, List[tk.Path]]] = None,
+    tf_fwd_input_name: str = "tf-fwd-input",
+) -> rasr.FlowNetwork:
+    """
+    Create the feature flow for a simple TF network that predicts frame-wise outputs, to be used
+    in combination with the `nn-precomputed-hybrid` feature-scorer setting in RASR.
+
+    The resulting flow is a trivial:
+
+        <link from="<tf_fwd_input_name>" to="tf-fwd:input"/>
+        <node name="tf-fwd" id="$(id)" filter="tensorflow-forward"/>
+        <link from="tf-fwd:log-posteriors" to="network:features"/>
+
+    With the config settings:
+
+        [flf-lattice-tool.network.recognizer.feature-extraction.tf-fwd.input-map.info-0]
+        param-name             = input
+        seq-length-tensor-name = extern_data/placeholders/<feature_tensor_name>/<feature_tensor_name>_dim0_size
+        tensor-name            = extern_data/placeholders/<feature_tensor_name>/<feature_tensor_name>
+
+        [flf-lattice-tool.network.recognizer.feature-extraction.tf-fwd.loader]
+        meta-graph-file    = <tf_graph>
+        required-libraries = <native_ops>
+        saved-model-file   = <tf_checkpoint>
+        type               = meta
+
+        [flf-lattice-tool.network.recognizer.feature-extraction.tf-fwd.output-map.info-0]
+        param-name  = <output_type>
+        tensor-name = <output_tensor_name>/output_batch_major
+
+
+    :param tf_graph: usually the output of a CompileTFGraphJob
+    :param tf_checkpoint: the checkpoint to load the model from, e.g. from a ReturnnTrainingJob or similar
+    :param feature_tensor_name: name of the extern data entry to feed the features to
+    :param output_layer_name: the name of the output layer containg log-probs,
+        it is expected that "<name>/output_batch_major" exists
```

Comment:
```suggestion
    :param output_layer_name: the name of the output layer, it is expected that 
        "<name>/output_batch_major" exists and returns log-probs.
```
just a suggestion for better wording imho.